frame/example-offchain-worker/src/lib.rs

```
@@ -385,14 +428,35 @@ impl<T: Trait> Module<T> {
 		// passing `price` as an argument.
 		let call = Call::submit_price_unsigned(block_number, price);
 
-		// Now let's create an unsigned transaction out of this call and submit it to the pool.
+		// Now let's create a transaction out of this call and submit it to the pool.
+		// Here we showcase two ways to send a transaction:
+		// 1. An unsigned transaction / unsigned payload (raw)
+		// 2. An unsigned transaction with a signed payload
+		//
 		// By default unsigned transactions are disallowed, so we need to whitelist this case
 		// by writing `UnsignedValidator`. Note that it's EXTREMELY important to carefuly
 		// implement unsigned validation logic, as any mistakes can lead to opening DoS or spam
 		// attack vectors. See validation logic docs for more details.
-		T::SubmitUnsignedTransaction::submit_unsigned(call)
-			.map_err(|()| "Unable to submit unsigned transaction.".into())
+		//
+		// Method 1: Unsigned transaction / Unsigned payload
+		let _result_raw: Result<(), &'static str> =
```

Please don't ignore the error, but rather propagate it upstream via `?`